doc/index.rst

```diff
@@ -82,6 +81,32 @@ Last but not least, daal4py allows :ref:`getting input data from streams <stream
         algo.compute(input)
     result = algo.finalize()
 
+oneAPI and GPU support in daal4py
+---------------------------------
+daal4py provide support of oneAPI conepts such as context and queues. This allow execution of
+algorithms on different devices, particularly GPUs. This implemented via 'with sycl_context("xpu")'
+blocks that will direct execution to selected device type. gpu, cpu and host devices are supported
+at the moment. Same aproach implemented for scikit-learn patching, so scikit-learn programs can be
+executed on GPU devices as well.   
+
+Intel CPU/GPU optimizations patching:
+```py
+from daal4py.sklearn import patch_sklearn
+from daal4py.oneapi import sycl_context
+patch_sklearn()
+
+from sklearn.svm import SVC
```

Comment:
I would suggest describing what algorithms to patch on the GPU. And choose another algorithm for an example, since now the svm is not patched for the GPU.

Reply:
changed to DBSCAN